snaplet-acid-state.cabal

```diff
@@ -25,11 +25,11 @@ Library
     Snap.Snaplet.AcidState
 
   build-depends:
-    acid-state                >= 0.6     && < 0.13,
+    acid-state                >= 0.6     && < 0.15,
```

Comment:
Using the latest (0.16) will trigger some compilation errors, so `0.15` is what we want to be using here (unless we want to patch the code).